spec/controllers/api/jobs_controller_spec.rb

```diff
@@ -11,6 +11,59 @@
         headers: {"Content-Type" => "application/json"}
       )
     get :index, params: search_params
-    assert response.ok?
+    body = JSON.parse(response.body)
+    expect(body[0]["company"]).to eq("VeriSign")
   end
+
+  it "wont save a job if your not logged in" do
+    post :create
+    body = JSON.parse(response.body)
+    expect(body["message"]).to eq("Please log in")
+  end
+
+  it "will create new job and save it if you are logged in" do
+    john = User.create!(name: "John", email: "John@johnny.com", password: "bro", password_confirmation: "bro", authorization_token: SecureRandom.hex(10))
+    job_key = { job: {jobkeys: "24c5d6db45db2b16" }}
+    stub_request(:get, /api.indeed.com/)
+      .to_return(
+        body: File.read(Rails.root.join("spec", "stubbed_requests", "single_job_search.json")),
+        headers: {"Content-Type" => "application/json"}
+      )
+      request.headers["HTTP_AUTHORIZATION"] = john.authorization_token
+      post :create, params: job_key
+      body = JSON.parse(response.body)
```

Comment:
Could we extract this out into into a method?   We have a lot of duplicated code in these tests?